### ISSUE TITLE
reserved identifier violation

### ISSUE THREAD

**Issue by elfring** [andlabs/libui#23]
I would like to point out that identifiers like "[`__LIBUI_UI_UNIX_H__`](https://github.com/andlabs/libui/blob/85ee2e93dd58663939530772841aeb97aa25e213/ui_unix.h#L7)" and "[`_UI_EXTERN`](https://github.com/andlabs/libui/blob/bf411e787e8c20d55e0351fe463e8619f226ddf5/ui.h#L16)" [do eventually not fit](https://www.securecoding.cert.org/confluence/display/cplusplus/DCL51-CPP.+Do+not+declare+or+define+a+reserved+identifier "Do not use identifiers that are reserved for the compiler implementation.") to the expected naming convention of the C++ language standard.
Would you like to adjust your selection for unique names?

**Comment by andlabs** [comment]
Yes, there's always a risk when anyone uses `_` at the beginning of a name for the private namespace. I suppose I could use a different prefix like `UI_I_` or something like that; not sure.

**Comment by elfring** [comment]
How do you think about to avoid that this software [depends on undefined behaviour](https://www.securecoding.cert.org/confluence/display/c/CC.+Undefined+Behavior#CC.UndefinedBehavior-ub_106 "Avoiding undefined behaviour?")?
